{
 "PRed": [
  {
   "PR": {
    "number": 1,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Compiler",
    "migrator_version": 0
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version"
   ]
  },
  {
   "PR": {
    "number": 2,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "CompilerRebuild",
    "migrator_version": 1,
    "name": "Python 3.7, GCC 7, R 3.5.1, openBLAS 0.3.2"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 3,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python38"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 4,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 2,
    "migrator_version": 0,
    "name": "python39"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "__lazy_json__": "pr_json/553188237.json"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "pypy"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "__lazy_json__": "pr_json/573784095.json"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "pypy37"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "__lazy_json__": "pr_json/779902825.json"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python310"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "labels": [
     {
      "name": "bot-rerun"
     }
    ],
    "number": 8,
    "state": "closed"
   },
   "data": {
    "bot_rerun": 1715545742.1044552,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python311"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "__lazy_json__": "pr_json/1545426127.json"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python312"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "__lazy_json__": "pr_json/1877788158.json"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "numpy2"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  }
 ],
 "bad": {
  "exception": "Error running check-solvable in container - error ValueError raised:\n\"Very odd pinning: pin_compatible('numpy')! Package numpy not found in host {}!\"",
  "traceback": [
   "Traceback (most recent call last):",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 573, in _run_migrator_on_feedstock_branch",
   "    migrator_uid, pr_json = run(",
   "                            ^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 366, in run",
   "    solvable, errors, _ = is_recipe_solvable(",
   "                          ^^^^^^^^^^^^^^^^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/solver_checks.py\", line 79, in is_recipe_solvable",
   "    return _is_recipe_solvable_containerized(",
   "           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/solver_checks.py\", line 138, in _is_recipe_solvable_containerized",
   "    data = run_container_task(",
   "           ^^^^^^^^^^^^^^^^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/utils.py\", line 273, in run_container_task",
   "    raise ContainerRuntimeError(",
   "conda_forge_tick.utils.ContainerRuntimeError: Error running check-solvable in container - error ValueError raised:",
   "\"Very odd pinning: pin_compatible('numpy')! Package numpy not found in host {}!\"",
   ""
  ]
 },
 "pinning_version": "2024.05.20.05.45.14",
 "pre_pr_migrator_attempts": {
  "pypy38": 69
 },
 "pre_pr_migrator_status": {
  "pypy38": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/9930635467\">bot CI job</a>): main:\nError running check-solvable in container - error ValueError raised:\n\"Very odd pinning: pin_compatible('numpy')! Package numpy not found in host {}!\""
 },
 "smithy_version": "3.36.0"
}